pkg/controllers/provisioning/provisioner.go

```diff
@@ -485,8 +486,7 @@ func (p *Provisioner) Validate(ctx context.Context, pod *corev1.Pod) error {
 func validateKarpenterManagedLabelCanExist(p *corev1.Pod) error {
 	for _, req := range scheduling.NewPodRequirements(p) {
 		if req.Key == v1.NodePoolLabelKey && req.Operator() == corev1.NodeSelectorOpDoesNotExist {
-			return fmt.Errorf("configured to not run on a Karpenter provisioned node via the %s %s requirement",
-				v1.NodePoolLabelKey, corev1.NodeSelectorOpDoesNotExist)
+			return serrors.Wrap(fmt.Errorf("configured to not run on a Karpenter provisioned node"), "requirement", fmt.Sprintf("%s %s", v1.NodePoolLabelKey, corev1.NodeSelectorOpDoesNotExist))
```

Comment:
```suggestion
			return serrors.Wrap(fmt.Errorf("configured not to run on a Karpenter provisioned node"), "requirement", fmt.Sprintf("%s %s", v1.NodePoolLabelKey, corev1.NodeSelectorOpDoesNotExist))
```

Reply:
This is tuh-mato, tomato to me -- I'd rather not change this just in case someone has a log query built on it